docs/updating/from_3.3/update_from_3.3.md

```diff
@@ -235,6 +235,34 @@ Run the following scripts:
     psql <database_name> < vendor/ibexa/installer/upgrade/db/postgresql/ibexa-3.3.8-to-3.3.9.sql
     ```
 
+### v3.3.14
+
+#### Update VCL
+
+Update your Varnish VCL file to align with [`docs/varnish/vcl/varnish5.vcl`](https://github.com/ezsystems/ezplatform-http-cache/blob/2.3/docs/varnish/vcl/varnish5.vcl).
+
+``` vcl hl_lines="4-7 16"
+// Compressing the content
+// ...
+
+// Modify xkey header to add translation suffix
+if (beresp.http.xkey && beresp.http.x-lang) {
+    set beresp.http.xkey = beresp.http.xkey + " " + regsuball(beresp.http.xkey, "(\S+)", "\1" + beresp.http.x-lang);
+}
+
+// ...
+
+if (client.ip ~ debuggers) {
+/// ...
+} else {
+    // Remove tag headers when delivering to non debug client
```

Comment:
```suggestion
    // Remove tag headers when delivering to non-debug client
```

Reply:
This is part of the pre-existing file: https://github.com/ezsystems/ezplatform-http-cache/blob/2.3/docs/varnish/vcl/varnish5.vcl#L329